### PR TITLE
enh(UI): Add new tile with the last ok status

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

As a user, I want to know for a resource not OK when was the last time it was OK.

**Fixes** # (issue)

A new tiles will be displayed for service in not OK state, nothing for resources in OK state.

Capture d’écran 2021-09-29 à 10 27 17

Capture d’écran 2021-09-30 à 11 32 12

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

Go to ressource status and selected resource with a non OK status, you can see now the new tile with informations about the last ok status for this resource.

### CODE REVIEW

File: www/front_src/src/Resources/Details/index.test.tsx [hunk 1]
```diff
@@ -522,7 +524,7 @@ describe(Details, () => {
     expect(getByText('10')).toBeInTheDocument();
     expect(getByText('CRITICAL')).toBeInTheDocument();
     expect(getByText('Centreon')).toBeInTheDocument();
-
+    expect(getByText(labelLastCheckWithOkStatus)).toBeInTheDocument();
```
Comment (bdauria) [inline www/front_src/src/Resources/Details/index.test.tsx:527]:
```suggestion
```

This is already stated below

**Review by bdauria — DISMISSED** [review]

**Review by Thebarda — DISMISSED** [review]

File: config/packages/serializer/Centreon/Monitoring.Resource.yml [hunk 2]
```diff
@@ -133,6 +133,10 @@ Centreon\Domain\Monitoring\Resource:
             type: DateTime<'Y-m-d\TH:i:sP'>
             groups:
                 - 'resource_main'
+              lastTimeWithNoIssue:
```
Comment (callapa) [inline config/packages/serializer/Centreon/Monitoring.Resource.yml:136]:
```suggestion
        lastTimeWithNoIssue:
```

**Review by callapa — CHANGES_REQUESTED** [review]

**Review by bdauria — DISMISSED** [review]

File: src/Centreon/Infrastructure/Monitoring/Resource/Provider/HostProvider.php [hunk 3]
```diff
@@ -153,6 +153,7 @@ private function prepareSubQuery(
             h.execution_time AS `execution_time`,
             h.latency AS `latency`,
             h.notify AS `notification_enabled`
```
Comment (kduret) [inline src/Centreon/Infrastructure/Monitoring/Resource/Provider/HostProvider.php:155]:
```suggestion
            h.notify AS `notification_enabled`,
```

**Review by Thebarda — DISMISSED** [review]

File: src/Centreon/Infrastructure/Monitoring/Resource/Provider/MetaServiceProvider.php [hunk 4]
```diff
@@ -152,6 +152,7 @@ private function prepareSubQuery(
             s.execution_time AS `execution_time`,
             s.latency AS `latency`,
             s.notify AS `notification_enabled`
```
Comment (kduret) [inline src/Centreon/Infrastructure/Monitoring/Resource/Provider/MetaServiceProvider.php:154]:
```suggestion
            s.notify AS `notification_enabled`,
```

File: src/Centreon/Infrastructure/Monitoring/Resource/Provider/ServiceProvider.php [hunk 5]
```diff
@@ -167,6 +167,7 @@ private function prepareSubQuery(
             s.execution_time AS `execution_time`,
             s.latency AS `latency`,
             s.notify AS `notification_enabled`
```
Comment (kduret) [inline src/Centreon/Infrastructure/Monitoring/Resource/Provider/ServiceProvider.php:169]:
```suggestion
            s.notify AS `notification_enabled`,
```

**Review by kduret — CHANGES_REQUESTED** [review]

**Review by kduret — APPROVED** [review]

**Review by callapa — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**